src/pages/BasketPage/index.js

```diff
@@ -12,7 +12,7 @@ import {
 
 import { BasketPageContent } from './components'
 import * as Styles2 from 'assets/StyledComponents/MainPageWrapper.css'
```

Comment:
Styles + importy zle pogrupowane

Reply:
mialem grupować na paczki absolute i relative nie rozumiem co jest zle

Reply:
W toku.

Reply:
- paczki npm ('redux', 'material-ui', etc.)

- absolute (assets/xxx, components/xxx, etc.)

- relative (../x, ../../x)

Reply:
linia 13 tutaj to relative ('./components'), linia 14 absolute ('assets/...')

Reply:
enemy spotted!
Done.